src/app/variables.ts

```diff
@@ -1,75 +1,48 @@
-import YAML from 'yaml'
+import { Env } from '@opendata-mvcr/assembly-line-shared'
```

Comment:
we agreed on the double-quotes, but this will be handled by another PR I expect.

Reply:
Yes!